sdks/python/apache_beam/io/gcp/gcsfilesystem.py

```diff
@@ -123,7 +123,7 @@ def _match(pattern, limit):
         pattern += '*'
       file_sizes = gcsio.GcsIO().size_of_files_in_glob(pattern, limit)
       metadata_list = [FileMetadata(path, size)
-                       for path, size in file_sizes.iteritems()]
+                       for path, size in file_sizes.items()]
```

Comment:
So in most places you seem to use `six.iteritems` it might be better to be consistent about that? It's possible that there could be a large number of input files as well in which case this would be less than great for Python 2.